### PR TITLE
[smoothscroll] update with new NPM name

### PR DESCRIPTION
**Pull request by galambalazs**

### CODE REVIEW

File: ajax/libs/smoothscroll/package.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "name": "smoothscroll",
+  "npmName": "smoothscroll-for-websites",
+  "npmFileMap": [ {
+    "basePath": "./",
```
Comment (PeterDaveHello) [inline ajax/libs/smoothscroll/package.json]: `"basePath": "",` will be easy here.

**Comment by PeterDaveHello** [comment]
Indent/format of package.json should be fixed.

**Comment by galambalazs** [comment]
updated

**Comment by PeterDaveHello** [comment]
hmmmm build failed, would you pelase run `node tools/fixFormat.js` to fix it? Thanks.

**Comment by galambalazs** [comment]
this is supposed to be valid json. What's so hard about parsing it?

**Comment by galambalazs** [comment]
When you have some time feel free to create a new PR with the correct syntax.

**Comment by PeterDaveHello** [comment]
Okay I'll take this.

**Comment by galambalazs** [comment]
thanks a lot Peter! :)

**Comment by PeterDaveHello** [comment]
No problem @galambalazs :+1:

**Pull request closed without merging**